src/ZString/Utf16ValueStringBuilder.cs

```diff
@@ -643,13 +642,13 @@ void AppendFormatInternal<T>(T arg, int width, ReadOnlySpan<char> format, string
                 if (typeof(T) == typeof(string))
                 {
                     var s = Unsafe.As<string>(arg);
-                    int padding = width - s.Length;
+                    int padding = width - s!.Length;
```

Comment:
Redundant when the initialization of `s` is adjusted - `var s = Unsafe.As<string>(arg)!;`